lib/src/chat/src/get_group_info.dart

```
@@ -0,0 +1,20 @@
+
+import '../../../push_restapi_dart.dart';
+
+Future<GroupInfoDTO?> getGroupInfo({required String chatId}) async {
+  if (chatId.isEmpty) {
+    throw Exception('chatId cannot be null or empty');
+  }
+
+  final result = await http.get(path: '/v2/chat/groups/$chatId');
+
+  if (result == null) {
```

We should replace 2 checks with a single if block and use || operator, also, when the result is null lets add a better error message according to the method like "Info not found" etc. Make check this for all the other methods, have found this for certain others as well. I dont think is a good idea to return null from an SDK method, instead lets throw an exception.